
This patch adds the missing PowerPC definitions (am I / are we the only
person / people on earth that build for PowerPC ?!?!?!?!?!?!?!?)

  - Rene Rebe <rene@rocklinux.org>

--- vnc_unixsrc/Xvnc/config/cf/Imake.cf.orig	2003-09-06 03:01:25.000000000 +0200
+++ vnc_unixsrc/Xvnc/config/cf/Imake.cf	2003-09-06 03:01:13.000000000 +0200
@@ -552,6 +552,10 @@
 #  define AlphaArchitecture
 #  undef __alpha
 # endif /* __alpha */
+# ifdef __powerpc
+#  define ppcArchitecture
+#  undef __powerpc
+# endif /* __powerpc */
 # ifdef mc68000
 #  define Mc68020Architecture
 #  undef mc68000
--- vnc_unixsrc/Xvnc/config/cf/linux.cf.orig	1998-05-11 18:38:06.000000000 +0200
+++ vnc_unixsrc/Xvnc/config/cf/linux.cf	2003-09-06 02:51:52.000000000 +0200
@@ -197,6 +197,13 @@
 #define AsmDefines		-D__ELF__
 #define CplusplusCmd		c++ -b alpha-linux
 #endif /* AlphaArchitecture */
+#ifdef ppcArchitecture
+#define CcCmd                   gcc
+#define AsCmd                   /usr/powerpc-linux/bin/as
+#define LdCmd                   ld -m elf32powerpc
+#define AsmDefines              -D__ELF__
+#define CplusplusCmd            c++ -b powerpc-linux
+#endif /* ppcArchitecture */
 #ifdef i386Architecture
 #define CcCmd			gcc -b i486-linux
 #define AsCmd			/usr/i486-linux/bin/as
@@ -252,6 +259,13 @@
 #define ServerExtraDefines	-DGCCUSESGAS XFree86ServerDefines -D_XSERVER64
 #endif /* AlphaArchitecture */
 
+#ifdef ppcArchitecture
+#define OptimizedCDebugFlags    -O2
+#define LinuxMachineDefines     -D__powerpc__
+#define ServerOSDefines         XFree86ServerOSDefines -DDDXTIME -DPART_NET
+#define ServerExtraDefines      -DGCCUSESGAS XFree86ServerDefines
+#endif /* ppcArchitecture */
+
 #ifdef Mc68020Architecture
 #define OptimizedCDebugFlags	-O2
 #define LinuxMachineDefines 	-D__mc68000__
